packages/compartment-mapper/src/import-hook.js

```diff
@@ -37,6 +37,26 @@ const has = (haystack, needle) => apply(hasOwnProperty, haystack, [needle]);
  */
 const resolveLocation = (rel, abs) => new URL(rel, abs).toString();
 
+/**
+ * @param {Error} error - error to throw on execute
+ * @returns {StaticModuleType}
+ */
+const postponeErrorToExecute = error => {
```

Comment:
I like “postpone”. You might like “defer” better. Your choice, regardless.

Reply:
Postpone feels to me like a more specific point in time than defer. I might be wrong. But I considered both and went for postpone as IMO better describing what's happening.

Reply:
used both where applicable. see new changes.